test/package.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "lisk-commander",
+	"oclif": {
+		"commands": "../src/commands",
+		"bin": "lisk",
+		"plugins": ["@oclif/plugin-help"]
+	},
+	"files": ["/bin", "/npm-shrinkwrap.json", "/oclif.manifest.json", "/dist"]
+}
```

Comment:
Is this file supposed to be here?

Reply:
This is needed for changing the directory of tests target to `src`
Without this file, it will tests against `dist` folder, which is defined in the parent `package.json`